components/tidb_query/src/rpn_expr/impl_cast.rs

```diff
@@ -323,6 +333,159 @@ fn cast_json_as_uint(ctx: &mut EvalContext, val: &Option<Json>) -> Result<Option
     }
 }
 
+// cast any as string, some cast functions reuse `cast_any_as_any`
+//
+// cast_json_as_string -> by cast_any_as_any<Json, String>
+
+#[rpn_fn(capture = [ctx, extra])]
+#[inline]
+fn cast_int_as_string(
```

Comment:
Can we define a macro to simplify this logic? (`cast_xxx_as_string` contains many duplicated logic)